build.gradle.kts

```diff
@@ -40,9 +36,9 @@ version = if (!project.hasProperty("useSnapshot") &&
 
     val suffix = ".${GitBasedVersioning.getGitCommitCount()}.${GitBasedVersioning.getGitShortCommitHash()}"
 
-    prefix + baseVersion + suffix
+    prefix + currentBranch + suffix
 } else {
-    "$baseVersion-SNAPSHOT"
+    "$currentBranch-SNAPSHOT"
```

Comment:
Is this change intended?

Reply:
I am sorry - I was not attentive and replaced the wrong variables. Will fix it all.